lib/docs/package.json

```diff
@@ -21,7 +21,9 @@
 		"@storybook/theming": "^6.2.9",
 		"autoprefixer": "^10.2.5",
 		"formik": "^2.2.7",
+		"lint": "^0.7.0",
 		"postcss": "^8.3.0",
+		"prettier": "^2.3.0",
```

Comment:
@NgocNhi123 I don't think that we need new packages here. Were you trying to run prettier and lint-fixing? If yes, you can run `yarn lint-fix` in the root directory.
https://github.com/moaijs/moai/blob/6095dc4cdac1aceddeb618d82414f8544c4d1e20/package.json#L6